app/policies/access_policy.rb

```diff
@@ -32,6 +33,7 @@ def configure
       can :read, Kithe::Model # whether publisehd or not
       can :update, Kithe::Model
 
+      can :see_admin_pages
```

Comment:
I don't _think_ you need to repeat `can :see_admin_pages` between multipe roles -- someone with the admin role should _also_ have the "lesser" roles. 

Let's see if we can get that to work, just have `can :see_admin_pages` once, at the least role that has it.

Reply:
You're right, no need to repeat it. Got rid of it.